### PR TITLE
Update a3p-integration after upgrade 16

### LINKED ISSUE #9835: update a3p-integration to post-upgrade16
## What is the Problem Being Solved?

As a final step in release 16 (this should get added to a runbook template), a3p-integration should be cleaned up to be ready for the next upgrade.

## Description of the Design

remove proposals and tests, and revert to a pristine `a:upgrade-next`.
 * update  to use new a3p image

## Security Considerations

Not directly a security issue.

## Scaling Considerations

None.

## Test Plan

remove tests.

## Upgrade Considerations

support ongoing upgrade management.
```[tasklist]
### Tasks
- [ ] https://github.com/Agoric/agoric-3-proposals/issues/167
```

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: Agoric/agoric-sdk#9835 
refs: Agoric/agoric-3-proposals#167

## Description

Remove changes that were included in [upgrade 16 ](https://github.com/Agoric/agoric-sdk/tree/agoric-upgrade-16-rc3)  from a3p-integration.

### Security Considerations

N/A

### Scaling Considerations

None

### Documentation Considerations
None

### Testing Considerations

Set up the test environment for the next phase.

### Upgrade Considerations

Clean up for the next set of upgrade tests.

### CODE REVIEW

File: a3p-integration/proposals/b:enable-orchestration/package.json
Comment (turadg) [inline a3p-integration/proposals/b:enable-orchestration/package.json]: this wasn't in u16

**Review by turadg — DISMISSED** [review]
`b:enable-orchestration` needs to stay. it wasn't part of u16. it's a separate proposal.

Regardless, `c:stake-bld` needn't be renamed.

File: a3p-integration/proposals/README.md
Comment (dckc) [inline a3p-integration/proposals/README.md]:
```suggestion
The proposals use `sdk-generate` to ensure that CI is always testing the latest version. But to publish we need a fixed version, to match the software upgrade that the BLD stakers decided on.
```

File: a3p-integration/proposals/README.md
Comment (Chris-Hibbert) [inline a3p-integration/proposals/README.md]:
```suggestion
`a3p-integration` holds draft proposals to test in agoric-sdk that they perform as expected. Once a proposal has been approved by BLD stakers it executes on chain and the proposal is no longer a draft. That requires moving it to the agoric-3-proposals repo to become part of the `latest` image.
```

Comment (Chris-Hibbert) [reply a3p-integration/proposals/README.md]: done

File: a3p-integration/proposals/README.md
Comment (Chris-Hibbert) [reply a3p-integration/proposals/README.md]: done

**Comment by Chris-Hibbert** [comment]
I reverted the removal of orchestration.

I will now wait for https://github.com/Agoric/agoric-3-proposals/pull/166 before merging, but a final review would be appreciated.

**Review by turadg — COMMENTED** [review]
I won't approve until this is based on `latest` that includes u16. Until then it's hard to say whether the changes are valid.

Weren't these in u16?
```
				// Enable low-level Orchestration.
				vm.CoreProposalStepForModules(
					"@agoric/builders/scripts/vats/init-network.js",
					"@agoric/builders/scripts/vats/init-localchain.js",
					"@agoric/builders/scripts/vats/init-transfer.js",
				),
```

**Comment by Chris-Hibbert** [comment]
> Weren't these in u16?

I removed those lines in upgrade.go.

I wasn't involved in orchestration. So some low level orchestration tooling was added in `upgrade.go`, but `b:enable-orchestration` was dropped. What did the latter contain? Were any tests for the orchestration tools run for U16?

**Comment by turadg** [comment]
> `b:enable-orchestration` was dropped. What did the latter contain?

Starting `vat-orchestration`

> Were any tests for the orchestration tools run for U16?

You mean the `init-network`, `init-localchain`, `init-transfer`?  Yes, there are other tests in upgrade-next. E.g. https://github.com/Agoric/agoric-sdk/blob/master/a3p-integration/proposals/a:upgrade-next/localchain.test.js#L34

You don't need to analyze those. They'll all start failing when this PR is on top of an A3P image with u16. It's blocked on that.

**Comment by Chris-Hibbert** [comment]
Rerunning now with the latest latest.

File: a3p-integration/proposals/README.md
Comment (turadg) [inline a3p-integration/proposals/README.md]: `fromTag`

File: a3p-integration/proposals/README.md
Comment (turadg) [inline a3p-integration/proposals/README.md:93]: I just noticed that some other docs are still pointing to `UNRELEASED_UPGRADE`, which wasn't updated in the docs when that string changed in https://github.com/Agoric/agoric-sdk/pull/9575

**Review by turadg — COMMENTED** [review]
[failing](https://github.com/Agoric/agoric-sdk/actions/runs/10396015703/job/28791782208?pr=9878) yet on,
```
#27 46.52 Waiting for upgrade height for UNRELEASED_A3P_INTEGRATION to be reached (need 1119, have 1118)
#27 47.44 10:47PM ERR UPGRADE "UNRELEASED_A3P_INTEGRATION" NEEDED at height: 1119: {"coreProposals":[]}
```

File: a3p-integration/proposals/README.md
Comment (dckc) [inline a3p-integration/proposals/README.md:33]: This results in files with absolute filenames on the developer's machine; for example `/home/connolly/.agoric/cache/b1-...`. Should those be changed? Some advice one way or the other would be nice here.

File: a3p-integration/proposals/a:upgrade-next/provisioning.test.js
Comment (dckc) [inline a3p-integration/proposals/a:upgrade-next/provisioning.test.js:34]: This seems like a feature that should work in every release going forward. Deleting the test seems unfortunate.

How about migrating it to the acceptance test?
 - #8961

**Review by dckc — COMMENTED** [review]
some suggestions to consider

I think approval should wait until it's passing ci.

**Review by turadg — APPROVED** [review]
I implemented some innocuous changes and cleaned up the commit history to get this in to unbreak master.

File: a3p-integration/proposals/a:upgrade-next/provisioning.test.js
Comment (turadg) [reply a3p-integration/proposals/a:upgrade-next/provisioning.test.js:34]: Punted to https://github.com/Agoric/agoric-sdk/issues/8961

**Comment by turadg** [comment]
@mergify update

**Comment by turadg** [comment]
@mergify rebase

**Pull request merged**
